compiler/rustc_const_eval/src/interpret/intrinsics.rs

```diff
@@ -457,7 +457,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 assert_eq!(input_len, dest_len, "Return vector length must match input length");
                 assert!(
                     index < dest_len,
-                    "Index `{}` must be in bounds of vector with length {}`",
+                    "Index `{}` must be in bounds of vector with length `{}`",
```

Comment:
If these are integers, it would be a bit odd to surround them with backticks, no?

Reply:
They are integers, and I agree that it is odd. It's done the same way a bit further down in the file. I didn't want to make changes that are too unrelated. I don't know, I can remove them as well.